src/components/Nav/NavData.js

```diff
@@ -0,0 +1,121 @@
+// 상수 데이터 1 : 헤더 좌측 메뉴
+const HEADER_LEFT_MENU = [
```

Comment:
```suggestion
export const HEADER_LEFT_MENU = [
```

로 할 수 있습니다. 그러면, 파일 하단에 있는것처럼 export를 따로 써주지 않아도 됩니다.